### PR TITLE
Add basic home view template

### PR DESCRIPTION
**Pull request by Dvessen83**
Had to remove the redundant application.css and also had to remove an unnecessary navbar partial (used in experiments) Fix Issue ID: #20

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1 +1,39 @@
-<h1>Home</h1>
+<body>
```
Comment (florestankorp) [inline app/views/pages/home.html.erb]: should not be in a body tag...

Comment (florestankorp) [inline app/views/pages/home.html.erb]: should not be in a body tag...

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -1 +1,39 @@
...
+                </div>
+                <footer>
+                    <p>&copy; 2017 Koalas, Inc.</p>
+                </footer>
+            </div>
+        </div>
+
+    </div>
+
+    <!-- Bootstrap core JavaScript
```
Comment (florestankorp) [inline app/views/pages/home.html.erb]: don't add your scripts in here...this is handled by the assets!

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (florestankorp) [inline app/views/pages/home.html.erb]: should not be wrapped in a body tag! this is handled by application.html.erb

File: app/views/pages/home.html.erb [hunk 3]
```diff
@@ -1 +1,39 @@
...
+    ================================================== -->
+    <!-- Placed at the end of the document so the pages load faster -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```
Comment (florestankorp) [inline app/views/pages/home.html.erb]: jQuery is shipped by default with our version of rails, no need to include! yay!

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (Dvessen83) [inline app/views/pages/home.html.erb]: removed body tag

File: app/views/pages/home.html.erb (hunk 2 above)
Comment (Dvessen83) [inline app/views/pages/home.html.erb]: script removed :)

File: app/views/pages/home.html.erb (hunk 3 above)
Comment (Dvessen83) [inline app/views/pages/home.html.erb]: Also removed

**Comment by Dvessen83** [comment]
#20

**Pull request merged**